test/fixtures/duplicate_filter.yaml

```diff
@@ -0,0 +1,27 @@
+kind: HTTPRoute
```

Comment:
Where are these fixtures used?

Reply:
These fixtures are just for manual testing at the moment, but I plan add them to  a more advanced e2e pipeline once I get the other parts of the webhook merged.

Reply:
I'd advise to add these fixtures later on then.
My concern is that we don't get to that e2e phase and then we have such files that no one remembers in the repo.

Reply:
Ack, will remove